it/uk/gov/hmrc/personaldetailsvalidation/services/PersonalDetailsService.scala

```diff
@@ -24,17 +31,11 @@ object PersonalDetailsService {
   }
 
   private def `POST /personal-details-validation`(personalDetails: PersonalDetails) = new {
+
     def toReturn(status: Int, header: (String, String)) =
       stubFor(
         post(urlEqualTo("/personal-details-validation"))
-          .withRequestBody(equalToJson(
-            s"""{
-               | "firstName":"${personalDetails.firstName}",
-               | "lastName":"${personalDetails.lastName}",
-               | "dateOfBirth":"${personalDetails.dateOfBirth.format(ISO_LOCAL_DATE)}",
-               | "nino":"${personalDetails.nino}"
-               |}
-               | """.stripMargin, true, false))
+          .withRequestBody(equalToJson(Json.toJson(personalDetails).toString()))
```

Comment:
Create two payloads of json into vals to make it visible then this can be matched  which then gets rid of the object and case class PersonalDetails.